python3/cloudwatch/src/lambda_function.py

```diff
@@ -117,11 +117,12 @@ def lambda_handler(event, context):
 
     logger.info("About to send {} logs".format(len(aws_logs_data['logEvents'])))
     for log in aws_logs_data['logEvents']:
-        if not isinstance(log, dict):
-            raise TypeError("Expected log inside logEvents to be a dict but found another type")
+        is_extra_data = log['message'].startswith('START') or log['message'].startswith('END') or log['message'].startswith('REPORT')
+        if not is_extra_data:
```

Comment:
@ronish31 turn it into a function and based on the return value decide if we want to ship the log or not